Gemfile.lock

```diff
@@ -32,6 +32,7 @@ GEM
       json (~> 1.4)
```

Comment:
On 6/5/2015 11:09 AM, sweir27 wrote:
>
> In Gemfile.lock 
> <https://github.com/sweir27/coop_gallery_213/pull/16#discussion_r31820645>:
>
> > @@ -32,6 +32,7 @@ GEM
> >        json (~> 1.4)
>
> Last thing, but could you please not commit this file either? It will 
> mess up the deployment with all of the windows-specific things in there.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/sweir27/coop_gallery_213/pull/16/files#r31820645>.
>
Sure, I'll add it to my .gitignore and I'll figure out how to remove it 
from the commit list.